### PR TITLE
Screen popup

### PR DESCRIPTION
**Pull request by Richez003**
popup design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,16 +8,29 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
```
Comment (brytebee) [inline index.html:10]: - [ ] **Optional:** Kindly note that commits messages should always be written as proper English sentences. This keeps your documentation and works uniform and professional.

image

File: index.html [hunk 2]
```diff
@@ -184,14 +176,72 @@ <h2><br>My Recent Works</h2>
...
 </div>
 <div class="button-cont">
-<button type="button" class="potfolio-but">See Project</button>
-</div>
-
+    <button type="button" class="potfolio-but">See Project</button>
+    </div>
+    
+    </div>
+    </div> -->
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** You have done an amazing job so far. However, the mobile view popup has some issues that need to be fixed.

popup

_**Hint**_
- You could reduce the height of the popup container.
- You can also truncate the text to a given length.
- You could hide the scroll `overflow`

File: index.html [hunk 3]
```diff
@@ -147,7 +139,7 @@ <h2><br>My Recent Works</h2>
 </div>
 </div>
 
-   
+
```
Comment (brytebee) [inline index.html]: - [x] **Required:** Kindly remove commented out code. It is unprofessional to push them.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, Team👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- No linter errors.✔️
- Desktop popup works excellent. ✔️
- Correctly used JS to dynamically populate the popup. ✔️
- Good handling and use of JS array of objects. ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Richez003) [reply index.html]: done that sir

File: index.html (hunk 2 above)
Comment (Richez003) [reply index.html]: I have reduced the height as requested .
truncated the text.
And hide scroll overflow.

**Review by topeogunleye — APPROVED** [review]
### Hi Team,
Great job on this project 👍🏻
# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! 
There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
>
### To Highlight
- [x]  Used GitHub Workflow :heavy_check_mark: 
>
> ## [Optional] suggestions:
> Please work on the mobile responsiveness of your website. Your project pop up is overflowing horizontally on Mobile:

https://user-images.githubusercontent.com/52934128/163592812-5573eab8-b078-4a1d-bca5-c76275865167.mp4
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**